datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -403,9 +426,9 @@ impl FileOpener for ParquetOpener {
         let projection = self.projection.clone();
         let pruning_predicate = self.pruning_predicate.clone();
         let table_schema = self.table_schema.clone();
-        let reorder_predicates = self.scan_options.reorder_predicates;
-        let pushdown_filters = self.scan_options.pushdown_filters;
-        let enable_page_index = self.scan_options.enable_page_index;
+        let reorder_predicates = self.reorder_filters;
```

Comment:
I also took the opportunity to change to consistently use the word `filters` rather than `filters` and `predciates`